### PR TITLE
#3170 Extend type assertion of API tests to be more informative

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3170)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by saivirtue** [comment]
/cc @oharastream/frontend 
I try to add some messages of each assertion for better error tracing.
Is that OK for you guys?
I will finish the rest API tests in this PR.

**Comment by chia7712** [comment]
Could we see both response and request via this PR?

**Comment by saivirtue** [comment]
> Could we see both response and request via this PR?

It's hard to say. Since the major purpose of this PR is to add more message of each "expect", i.e., the log or console won't get print out during QA.
I want to add the req/res data either, but it should be done in another task of #3170

**Comment by konekoya** [comment]
> I try to add some messages of each assertion for better error tracing.
Is that OK for you guys?
I will finish the rest API tests in this PR.

The only concern here is it's going to be quite verbose when writing these tests. If Cypress (or mocha) provides a better assert error output, then we won't have to add this manually...

I will try to whip up an example and file an issue over Cypress' repo and see if they can help us out :)

File: ohara-manager/client/cypress/configs/cypress.api.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "integrationFolder": "cypress/api",
   "reporterOptions": {
     "mochaFile": "../test-reports/apiReport-[hash].xml",
-    "toConsole": true
+    "toConsole": true,
+    "jenkinsMode": true
```
Comment (saivirtue) [inline ohara-manager/client/cypress/configs/cypress.api.json]: from [mocha-junit-reporter](https://github.com/michaelleeallen/mocha-junit-reporter#full-configuration-options):
> if set to truthy value will return xml that will display nice results in Jenkins

Comment (konekoya) [reply ohara-manager/client/cypress/configs/cypress.api.json]: @saivirtue Does this config affect the output report?

Comment (saivirtue) [reply ohara-manager/client/cypress/configs/cypress.api.json]: The most obvious changes are the `name` and `className`:

**before**
```xml
<testcase name="Broker API createBroker" time="0.897" classname="createBroker">
```

**after**
```xml
<testcase name="createBroker" time="4.481" classname="Broker API">
```

Comment (saivirtue) [reply ohara-manager/client/cypress/configs/cypress.api.json]: But it seems not to break our Jenkins behavior, I thought we could add this config?
ping @chia7712

**Pull request merged**